core/api_routes/agents.py

```
@@ -0,0 +1,1212 @@
+        logger.error(f"Error listing agents: {str(e)}")
+        return JSONResponse(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            content=error_response(
+                message="Failed to list agents",
+                code="AGENT_LIST_ERROR",
+                details={"error": str(e)}
+            )
+        )
+
+@router.get("/{agent_id}", response_model=Dict[str, Any])
+async def get_agent(agent_id: str) -> JSONResponse:
```

Ruff (0.8.2)

Redefinition of unused `get_agent` from line 20

(F811)